mininet/simpletest.py

```diff
@@ -15,6 +15,7 @@
 
 from os import path
 from os import makedirs
+from datacollector import DataCollector
 
 HOME_FOLDER = os.getenv('HOME')
 LOG_PATH = HOME_FOLDER + '/TARNProject/TARN/logs/'
```

Comment:
Maybe we should consider just adding a `logs` directory in the `TARN` directory, so we don't have to create a separate `/TARNProject/TARN/logs`.